urad-rs-za-makroekonomske-analize-in-razvoj.csl

```diff
@@ -823,9 +829,13 @@
                     </group>
                   </if>
                   <else>
-                    <group>
-                      <text variable="genre"/>
-                      <text variable="event-place" prefix=", "/>
+                    <group delimiter=", " prefix=", ">
+                      <choose>
+                        <if match="any" variable="genre">
```

Comment:
The test here is unnecessary. text variable implicitly test for the variable's presence